### DISCUSSION TITLE
青龙使用了别的京东脚本后导致冲突

### DISCUSSION THREAD

**Discussion by cnqv** [RayWangQvQ/BiliBiliToolPro#750] [Q&A]
开始执行后提示：
**_cp: cannot create regular file '/ql/data/scripts/DefaultTasks/CK_WxPusherUid.json': No such file or directory_**
bilitool: 青龙repo目录: /ql/data/repo
bilitool: bili仓库目录: /ql/data/repo/RayWangQvQ_BiliBiliToolPro
bilitool: 当前系统发行版本：alpine.3.20.1
bilitool: 当前系统：linux-musl
bilitool: 当前架构：x64
bilitool: 当前选择的运行方式：dotnet
bilitool: 已安装dotnet，当前版本：6.0.132
bilitool: which dotnet: /usr/bin/dotnet
bilitool: 环境正常，本次无需安装

但是可以正常运行，运行结束后提示：
[08:00:23 INF] 运行环境：Production
[08:00:23 INF] 应用目录：/ql/data/repo/RayWangQvQ_BiliBiliToolPro/src/Ray.BiliBiliTool.Console/bin/Debug/net6.0/
[08:00:23 INF] 运行结束
**_/ql/shell/share.sh: line 476: task_after: unbound variable_**
最后提示错误，导致任务不会自行停止，一直卡在运行中，请问如何解决？

**Comment by SingTsang** [comment]
一样的问题，等待大佬的解决办法

**Comment by RayWangQvQ** [comment]
cp: cannot create regular file '/ql/data/scripts/DefaultTasks/CK_WxPusherUid.json': No such file or directory
/ql/shell/share.sh: line 476: task_after: unbound variable

看起来与当前项目无关的

**Reply by cnqv** [reply]
求教大佬，这种情况有办法解决吗？

**Comment by Ling128** [comment]
same issue

**Comment by starsink-d** [comment]
同样的问题
